plugins/usbdmx/DMXCProjectsNodleU1Factory.cpp

```diff
@@ -30,14 +30,25 @@ namespace ola {
 namespace plugin {
 namespace usbdmx {
 
-const uint16_t DMXCProjectsNodleU1Factory::VENDOR_ID = 0x16d0;
-const uint16_t DMXCProjectsNodleU1Factory::PRODUCT_ID = 0x0830;
+const uint16_t DMXCProjectsNodleU1Factory::VENDOR_ID_NOODLE_U1 = 0x16d0;
```

Comment:
Can we maybe have VENDOR_ID_DMXCONTROL_PROJECTS or similar.

Reply:
So to get you right, @peternewman, you want a line like this?
`const uint16_t DMXCProjectsNodleU1Factory::VENDOR_ID_DMXCONTROL_PROJECTS = 0x16d0;`

I think that's quite long but would fit better the naming of the other two (DE and FX5) as both are the "producers" of the interface and not the interface itself.

Reply:
BTW: This breaks linting as the line get's to long.

Reply:
Done in 30577e3cc69a8acf161f5f1e6dd6a8e617d9b704